hpedockerplugin/volume_manager.py

```diff
@@ -1354,3 +1384,37 @@ def _get_required_qos_field(qos_detail):
         qos_filter['vvset_name'] = qos_detail['name']
 
         return qos_filter
+
+    def _decrypt_password(self, hpepluginconfig):
```

Comment:
Will this method definition work correctly in a case, where the plugin is upgraded from a previous release, and now the _decrypt_password() is called on a plain text ?

Reply:
Made the change in function to work with previous release as well